src/flows/recommendation_api/corpus_candidate_sets/recommended_by_recency.py

```diff
@@ -0,0 +1,49 @@
+from prefect import Flow
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    create_corpus_candidate_set_record,
+    load_feature_record,
+    feature_group,
+    validate_corpus_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+
+FLOW_NAME = get_flow_name(__file__)
+
+RECENT_RECOMMENDED_CANDIDATE_SET_ID = "2066c835-a940-45ec-b1f7-267457d9e0a2"
+
+EXPORT_RECOMMENDED_CANDIDATE_SET_SQL = """
+SELECT 
+    approved_corpus_item_external_id as "ID", 
+    topic as "TOPIC"
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS"
+WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD("day", %(MAX_AGE_DAYS)s, current_timestamp())
+AND CORPUS_REVIEW_STATUS = 'recommendation'
+AND LANGUAGE = 'EN'
```

Comment:
do we want `is_syndicated = FALSE` or `is_collection = FALSE`?

Reply:
[Content requirements doc](https://docs.google.com/document/d/15gBxnXlsKTgGyjZe72VUc43aK9ZoAArP9NaJDNLIaps/edit#heading=h.mljabq907ibf) do not specify that, so I'm assume we include everything. Easy enough to change later if requirements are added.